chart-infra/templates/scale-to-zero.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if (eq .Values.kubernetes.env "staging") }}
+# workerScaleToZero isn't picked up - needs to be in master?
+{{- if or (eq .Values.myAccount.workerScaleToZero "true") (eq .Values.kubernetes.env "staging") }}
```

Comment:
`.Values.myAccount.workerScaleToZero` doesn't seem to be transmitted from `iac` correctly. Current guess is that `iac` changes need to be in master. May also be configured incorrectly